### ISSUE TITLE
Python deepspeech bindings still looking for CUDA 9 when dependencies are CUDA 10 

### ISSUE THREAD

**Issue by PoLabs** [mozilla/DeepSpeech#2102]
Hi, hoping this one isn't too beat to death, but I'm getting the 'can't find libcusolver.so.90' error. I have CUDA 10 installed , not CUDA 9, in accordance with the dependencies: https://github.com/mozilla/DeepSpeech#cuda-dependency . I installed the python bindings via 'pip3 install deepspeech-gpu'. Found a suggestion for newer deepspeech version in the issues and tried 'pip3 install deepspeech 0.5.0A7' but it has the same error.

`$ deepspeech --model models/output_graph.pbmm --alphabet models/alphabet.txt --lm models/lm.binary --trie models/trie --audio sample-audio/NPR-science-podcast.wav
`
```
Traceback (most recent call last):
  File "/home/polabs1/.local/lib/python3.5/site-packages/deepspeech/impl.py", line 14, in swig_import_helper
    return importlib.import_module(mname)
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 666, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 577, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 906, in create_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
ImportError: libcusolver.so.9.0: cannot open shared object file: No such file or directory

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/polabs1/.local/bin/deepspeech", line 7, in <module>
    from deepspeech.client import main
  File "/home/polabs1/.local/lib/python3.5/site-packages/deepspeech/__init__.py", line 4, in <module>
    from deepspeech.impl import AudioToInputVector as audioToInputVector
  File "/home/polabs1/.local/lib/python3.5/site-packages/deepspeech/impl.py", line 17, in <module>
    _impl = swig_import_helper()
  File "/home/polabs1/.local/lib/python3.5/site-packages/deepspeech/impl.py", line 16, in swig_import_helper
    return importlib.import_module('_impl')
  File "/usr/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ImportError: No module named '_impl'
```

My system: ubuntu  16, cuda 10, cudnn 7.4.1, tensorflow-gpu 1.13.1, deepspeech 0.5.0A7 on a RTX 2070. The libcusolver.so.9.0 file doesn't exist on my system, do I need to have cuda 9 installed alongside cuda 10 in some way? Or some library pointer from cuda9 requirements to the cuda10? Believe I am forced into cuda 10 due to the RTX 2070.

```
$ cat /usr/local/cuda/include/cudnn.h | grep CUDNN_MAJOR -A 2
#define CUDNN_MAJOR 7
#define CUDNN_MINOR 4
#define CUDNN_PATCHLEVEL 1
--
#define CUDNN_VERSION (CUDNN_MAJOR * 1000 + CUDNN_MINOR * 100 + CUDNN_PATCHLEVEL)

#include "driver_types.h"
```
Is this cudnn version wrong? I've installed 7.5 but thought it would show 'minor' as 4.

```
$ nvcc --version
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2018 NVIDIA Corporation
Built on Sat_Aug_25_21:08:01_CDT_2018
Cuda compilation tools, release 10.0, V10.0.130
```

Thanks in advance!

**Comment by kdavis-mozilla** [comment]
Did you un-install deepspeech-gpu before installing deepspeech 0.5.0A7 over it?

Also, there is no model released for any 0.5.0A* version so using the 0.4.1 model, which I assume output_graph.pbmm is, with 0.5.0A7 will not work.

I'd suggest using the 0.4.1 deepspeech-gpu and referring to the [0.4.1. documents](https://github.com/mozilla/DeepSpeech/tree/v0.4.1).

**Comment by PoLabs** [comment]
Ok thank you for the quick response. Can't run cuda9 on my RTX card so I'll have to look for another speech2text model. Closing for now and will revisit if/when there's 0.5 models
